### DISCUSSION TITLE
请问不支持ipv6吗，netstat看没有监听tcp6，实测也不支持

### DISCUSSION THREAD

**Discussion by zhendery** [messense/aliyundrive-webdav#361] [Q&A] [answered]
### 问题描述

请问不支持ipv6吗

### 重现步骤

请问不支持ipv6吗，netstat看没有监听tcp6，实测也不支持

### 版本

amd64

### 运行平台

linux

### 日志

无

**Comment by messense** [comment, marked as answer]
`—host ::`

**Reply by messense** [reply]
https://github.com/messense/aliyundrive-webdav/discussions?discussions_q=ipv6

学会搜索，不要问重复的问题浪费时间。

**Reply by zhendery** [reply]
啊这，感谢，在issues里没找到，竟然在discussions里
抱歉打扰了
